### PR TITLE
zh-translation:/docs/reference/config/analysis/message-format/index.md

### PR DESCRIPTION
**Pull request by ghl1024**
ref: [servicemesher/istio-official-translation/issues/1491](https://github.com/servicemesher/istio-official-translation/issues/1491)

### CODE REVIEW

**Comment by ghl1024** [comment]
@tanjunchen
麻烦闲了帮我review下，谢谢。

File: content/zh/docs/reference/config/analysis/message-format/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Analyzer Message Format
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/analysis/message-format/index.md]: 未译

File: content/zh/docs/reference/config/analysis/message-format/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+The `istioctl analyze` command provides messages in the format:
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/analysis/message-format/index.md]: 去除英文

File: content/zh/docs/reference/config/analysis/message-format/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+该 `istioctl analyze` 命令规定以下格式：
+
+{{< text plain >}}
+<level> [<code>] (<affected-resource>) <message-details>
+{{< /text >}}
+
+The `<affected-resource>` field expands to:
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/analysis/message-format/index.md]: 未译

File: content/zh/docs/reference/config/analysis/message-format/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+{{< text plain >}}
+<resource-kind> <resource-name>.<resource-namespace>
+{{< /text >}}
+
+例如:
+
+{{< text plain >}}
+Error [IST0101] (VirtualService httpbin.default) Referenced gateway not found: "httpbin-gateway-bogus"
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/analysis/message-format/index.md]: 未译

File: content/zh/docs/reference/config/analysis/message-format/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+{{< /text >}}
+
+The `<message-details>` field contains a detailed description that may contain further information to help resolve the problem. The namespace suffix is omitted for cluster-scoped resources, for example `namespace`.
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/analysis/message-format/index.md]: 未译

**Comment by ghl1024** [comment]
@tanjunchen

File: content/zh/docs/reference/config/analysis/message-format/index.md (hunk 4 above)
Comment (tanjunchen) [reply content/zh/docs/reference/config/analysis/message-format/index.md]: Error [IST0101] (VirtualService httpbin.default) Referenced gateway not found: "httpbin-gateway-bogus"
->
错误 [IST0101] (VirtualService httpbin.default) 找不到相关的 gateway："httpbin-gateway-bogus"

**Comment by tanjunchen** [comment]
@ghl1024 fix conflict

**Pull request closed without merging**